app/src/main/java/no/usn/mob3000/ui/components/settings/SettingsSectionApplication.kt

```diff
@@ -114,6 +116,7 @@ fun SettingsSectionApplication(
             onValueChange = {},
             readOnly = true,
             trailingIcon = { ExposedDropdownMenuDefaults.TrailingIcon(expanded = themeExpanded) },
+            textStyle = TextStyle(color = MaterialTheme.colorScheme.onBackground),
             modifier = Modifier.menuAnchor(MenuAnchorType.PrimaryNotEditable)
```

Comment:
[See this comment](https://github.com/frigvid/mob3000-group11/pull/27/files#r1847123849).